server/rating.py

```diff
@@ -10,36 +10,57 @@ class RatingType(Enum):
     GLOBAL = "global"
     LADDER_1V1 = "ladder_1v1"
 
+    def __eq__(self, other):
+        if isinstance(other, str):
+            return self.value == other
+        else:
+            return super().__eq__(other)
 
-class RatingTypeMap(MutableMapping):
-    def __init__(self, default):
-        MutableMapping.__init__(self)
-        self._back = {}
+    def __hash__(self):
+        return hash(self.value)
 
-        for rtype in RatingType:
-            self[rtype] = default
 
-    def __getitem__(self, key: RatingType):
-        return self._back[key]
+K = Union[RatingType, str]
+V = TypeVar("V")
```

Comment:
I'd choose better naming. K, V are first letters of my first name and name after father (it's third name that russians are using) :smile:

Reply:
IMO these are totally standard names for key/value in templates.

Reply:
It's like calling your loop counter `i` instead of `index`